src/PageHead.jsx

```diff
@@ -52,6 +60,9 @@ class PageHead extends React.Component {
 				>
 					<FlexItem>
 						<Chunk className='align--center atMedium_align--left'>
+							{titleLabel &&
+								<p className={`${titleLabelClassNames} text--label`}>{titleLabel}</p>
```

Comment:
this is probably my bad when looking at the example on line 68. But this class should actually be a part of the `titleLabelClassNames` like below.

```
		const titleLabelClassNames = cx(
			PAGE_TITLELABEL_CLASS,
			'text--secondary',
			titleLabelClass
		);
```